lib/clients/web/facets/team.js

```diff
@@ -21,8 +21,6 @@ function TeamFacet(makeAPICall) {
  * @see {@link https://api.slack.com/methods/team.accessLogs|team.accessLogs}
  *
  * @param {Object=} opts
-
-
```

Comment:
There are a lot of whitespace changes in this PR…would you mind adjusting that? Just to keep things clean?